bodhi/server/push.py

```diff
@@ -71,19 +72,27 @@ def push(username, password, cert_prefix, **kwargs):
                 updates.append(update)
                 click.echo(update)
     else:
+        config_uri = '/etc/bodhi/production.ini'
```

Comment:
It might be nice to make this non-hardcoded, especially if we want to be able to unit test this module.